test/main_test.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:battle_it_out/persistence/character.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() async {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  File file = File('assets/test/character_test.json');
+  Character character = await Character.create(jsonDecode(await file.readAsString()));
```

Comment:
these variables should be declared in each test so that the tests don't depend on each other, preferably in a function